index.html

```diff
@@ -26,8 +25,163 @@ <h1>  Hey there. I’m Mariane <span class="green" >I’m a software developer</
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
+                    <div class="butt"><button class="btn" type="button"> See Project</button></div>
+                </article>
+                <article class="project">
+                    <div class ="image-placeholder"></div>
+                    <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+                    <ul class ="lang">
+                        <li class="language">html</li>
+                        <li class="language">css</li>
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
+                    <div class="butt"><button class="btn" type="button"> See Project</button></div>
+                </article> 
+                <article class="project">
+                    <div class ="image-placeholder"></div>
+                    <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+                    <ul class ="lang">
+                        <li class="language">html</li>
+                        <li class="language">css</li>
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
+                    <div class="butt"><button class="btn" type="button"> See Project</button></div>
+                </article>
+                 <article class="project">
+                    <div class ="image-placeholder"></div>
+                    <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+                    <ul class ="lang">
+                        <li class="language">html</li>
+                        <li class="language">css</li>
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
+                    <div class="butt"><button class="btn" type="button"> See Project</button></div>
+                </article>
+            </div>
+            </section>
+            <section>
+                <div class="about2">
+                    <h2>About me</h2>
+
+                <div class="about">
+                </div>
+                <div class="about-text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+                </div>
+            </div>
+                <div class="butt resume-container"><button class="btn btn-resume" type="button"> Get My Resume</button></div>
+                
+                <div class="skills-container">
+                <article class="project2">
+                <div class="language-icon"> <h><img src="/images/icon-languages (1).png" alt=""></h></div>
```

Comment:
- [ ] In each of your project cards, you have used a tag `<h>` which as far as I know doesn't exist. Even if it may exist, it has nothing to do with the image. Please remove it and stick with the image <img> tag only 😃 👍 
- [ ] I have realized that most of your images are stored in the images folder which is inside another images folder which I believe that it is the reason why you got mistaken here. For example: the correct path for the above image would be `./images/images/icon-languages (1).png`
👇 👇 
![image](https://user-images.githubusercontent.com/90222110/192061904-47161345-97fa-4412-a35b-3c199a5801cb.png)